lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+class Menu
+
+  attr_reader :menu
+
+  def initialize
+
```

Comment:
It probably would have been better to write some code to import different menus.